src/pieces.rs

```diff
@@ -65,15 +74,17 @@ impl AddAssign<Move> for Position {
 // Add unit tests at the bottom of each file. Each tests module should only have access to super (non integration)
 #[cfg(test)]
 mod tests {
-    use super::Position;
+    use super::*;
```

Comment:
Nice, so this imports all of our structs basically?

Reply:
`super` in this case would be `piece` and so includes everything in the `piece` module